src/CattyTests/Setup/Firebase/FirebaseCrashlyticsSetupTests.swift

```diff
@@ -167,4 +167,27 @@ final class FirebaseCrashlyticsSetupTests: XCTestCase {
         XCTAssertEqual(0, crashlytics!.logs.count)
         XCTAssertEqual(1, crashlytics!.records.count)
     }
+
+    func testProjectSearchFailureNotification() {
+        UserDefaults.standard.set(true, forKey: kFirebaseSendCrashReports)
+        app?.setupCrashlytics()
+
+        let errorInfo = ProjectFetchFailureInfo(url: "testUrl", statusCode: nil, description: "Invalid API response while fetching project", projectName: nil)
+
+        let expectedErrorValue = "No value"
+        let projectInfo: [String: Any] = ["description": errorInfo.description,
+                                          "projectName": expectedErrorValue,
```

Comment:
Great that you tested the expected error value in case of nil. However, it is much more important to test the real values. So I would suggestion to either add another test method testing the actual _projectName_ and _statusCode_ or change it here directly.

Reply:
I have made the changes please review.

Reply:
Sorry, I can not see any changes. This test does still not verify whether the proper `projectName` and `statusCode` are set, i.e. it will succeed also when the actual values do not get set in the method under test. Thanks!

Reply:
@m-herold have you checked lines 192 to 205, I think that will solve the problem you pointed. 

Please correct me if I am wrong.

Reply:
Thanks, I did not see those lines.